components/gitpod-db/BUILD.yaml

```diff
@@ -64,7 +64,11 @@ packages:
     ephemeral: true
     config:
       commands:
+        # Check if a DB is present. If not: start one and wait until it's up
```

Comment:
Would be nice to include some of the description from the PR here, e.g. that the DB is present in werft jobs as it is running as a sidebar